python/tank_vendor/shotgun_authentication/defaults_manager.py

```diff
@@ -32,22 +42,36 @@ def __init__(self):
 
     def is_host_fixed(self):
         """
-        When doing an interactive login, this indicates if the user can choose
-        the host to connect to.
-
-        :returns: True is the host can't be edited, False otherwise,
+        When doing an interactive login, this indicates if the user can decide
+        the host to connect to. In its default implementation, the DefaultsManager
+        will indicate that the host is not fixed, meaning that the user will be
+        presented with an option to pick a site at login time.
+        
+        With something like Toolkit, where each project already have a specific
+        project association, you typically want to override this to return True,
```

Comment:
I think you mean "site association", right? A project is attached to a Shotgun site.